engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/log/NadelLog.kt

```diff
@@ -0,0 +1,51 @@
+package graphql.nadel.enginekt.log
+
+import graphql.nadel.util.LogKit
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+internal inline fun <reified T> getLogger(): Logger {
```

Comment:
Sorry slightly unrelated, I worked off a branch I where I started adding debug logging to investigate the issue. Keeping these in because they're useful.

We need to improve logging too:

* We need a StructuredArgsLogger like in the AGG and RB, we aren't logging the request ID or execution ID along with every request and I had to guess which logs matched which requests based on anomalous data/times.
* Maybe others